SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -410,9 +419,9 @@ - (NSOperationQueue *)sendingQueueForMessage:(TSOutgoingMessage *)message
     }
 }
 
-- (void)sendMessage:(TSOutgoingMessage *)message
-            success:(void (^)(void))successHandler
-            failure:(void (^)(NSError *error))failureHandler
+- (void)enqueueOutgoingMessage:(TSOutgoingMessage *)message
+                       success:(void (^)(void))successHandler
+                       failure:(void (^)(NSError *error))failureHandler
```

Comment:
I approve of these name changes!

TIOLI: we could safely omit the word "outgoing" for brevity, since you don't send "incoming" messages

Reply:
Done.